### ISSUE TITLE
Consider creating an independent "Resources" website

### ISSUE THREAD

**Issue by jvican** [scala/scala-lang#614]
The Scala website has a section for learning resources: http://scala-lang.org/community/index.html#community-libraries-and-tools.
  
However, I think it deserves its own independent webpage. People visiting the website may miss "Community-Powered Learning Resources" and "Community Libraries and Tools". These actions have value on their own and they are actually quite critical for developers to know about.
  
What I propose is to have an independent page that we can use to let people know all the learning resources/tools they can use to code in Scala. That would also include courses as Coursera and the ones reported by @MarkCLewis in [this Scala Contributors thread](https://contributors.scala-lang.org/t/total-beginner-to-programming-scala-what-would-you-suggest-to-get-them-started/450/20?u=jvican). I'm sure there are more to add, too.
  
We may also want to add more learning resouces, like Scala books, good blog articles (e.g. Li Haoyi's blog posts are great and very idiomatic), link to previous Scala Lang blog articles that are useful for learning, etc.
  
I also add to the discussion my intention to add a section to get people started to code in Scala. I'd like to have links to labelled issues in projects that people can work on. We could have community projects like the Scala Platform modules (better-files, json AST, possibly enumeratum), this website itself, sbt, and in general any other project that enjoys of maintainers that are eager to help and welcome new contributors.
  
The motivation of this last section is to let people get their hands dirty and hack on Scala from day one. I think it makes a difference having some guidance on far-reaching OSS projects that could motivate their learning experience while benefitting the whole Scala community.
  
What do you think @heathermiller @SethTisue @adriaanm ?

**Comment by adriaanm** [comment]
Yes! This is an important topic. What are website visitors looking for when they go to scala-lang? 

Right now we break it down as:
  - documentation
  - download
  - community
  - contribute

You've identified two more important needs/interests:
  - learn (which is more than docs, it could be a course, a forum, a collection of blogs)
  - get involved

Here's another way to break it down (kind of according to increasing levels of experience):
  - get started (tutorial/tour, scalafiddle to try without downloading, first-timer install instructions)
  - download (returning customers who just want the bits)
  - docs (same as now, for quick reference: scaladocs, lang spec)
  - learning (if you want to deepen your knowledge, you wouldn't click on get started, and docs sounds too static)
  - get involved (combination of community + contribute)

**Comment by tpolecat** [comment]
"Can you recommend learning resources?" is a very frequent question on Gitter, IRC, and Reddit, so 💯 for amplifying this part of the website.

**Comment by travis032654** [comment]
What if "Getting Started" was on the learning page? I like to minimize the number of choices of things to click on. I think it makes it easier to navigate.

**Comment by jvican** [comment]
>  Many of the resources assume people have several years of development in Java or perhaps some other language. Trying to make that clear will go a long way to help people determine which resource is best for them.
  
Totally agreed.
  
> Along those lines, saying what version of Scala it was created for could also be useful. This is a very dynamic ecosystem.
  
Good idea, we can also write a note that asks readers to report if material has become obsolete. I think that sometimes not even we know it is.
  
> What if "Getting Started" was on the learning page? I like to minimize the number of choices of things to click on. I think it makes it easier to navigate.
  
Sure, this works for me. I would need to see the final look and structure of the website, though, because putting it everything into "Getting Started" would divert the focus of the sections. I would prefer short and accessible sections rather than long and detailed ones.
  
@travis032654 Would you be interested in getting a draft of all this?

**Comment by travis032654** [comment]
Sure!

**Comment by jvican** [comment]
@travissarles Can you have a look at this any time soon? It would be great to have this and optimize for contributors :smile:.

**Comment by xingyif** [comment]
FYI, scalabridge has a [resources](https://scalabridge.gitbooks.io/curriculum/content/resources.html) page that might be useful. It separates available resources by levels: Introductory, intermediate, and advanced, and also directly link to them.

**Comment by SethTisue** [comment]
I agree that the [Community page](https://scala-lang.org/community/) is too long, and a split into Community and Resources seems like a logical split. At least, I can't think of a better one
